Pod/Cache/GenericCoreDataStack.swift

```diff
@@ -108,52 +108,45 @@ public class GenericCoreDataStack<CoordinatorType: NSPersistentStoreCoordinator,
                 
-                let storeURL = cachesURL.URLByAppendingPathComponent("\(storeNamePrefix).sqlite")
+            } else {
+                try self.addPersistentStore(NSSQLiteStoreType, configuration: nil, URL: storeURL, options: nil, migrationManger: nil)
+            }
+        } else {
+            for configuration in configurations {
                 
-                if let (storeType, storeOptions, migrationManager) = options[defaultModelConfigurationName] {
-                    try self.addPersistentStore(storeType, configuration: nil, URL: storeURL, options: storeOptions, migrationManger: migrationManager)
+                if configuration != defaultModelConfigurationName {
                     
-                } else {
-                    try self.addPersistentStore(NSSQLiteStoreType, configuration: nil, URL: storeURL, options: nil, migrationManger: nil)
-                }
-            } else {
-                for configuration in configurations {
+                    let storeURL = cachesURL.URLByAppendingPathComponent("\(storeNamePrefix)\(configuration).sqlite")
                     
-                    if configuration != defaultModelConfigurationName {
+                    if let (storeType, storeOptions, migrationManager) = options[configuration] {
+                        try self.addPersistentStore(storeType, configuration: configuration, URL: storeURL, options: storeOptions, migrationManger: migrationManager)
                         
-                        let storeURL = cachesURL.URLByAppendingPathComponent("\(storeNamePrefix)\(configuration).sqlite")
-                        
-                        if let (storeType, storeOptions, migrationManager) = options[configuration] {
-                            try self.addPersistentStore(storeType, configuration: configuration, URL: storeURL, options: storeOptions, migrationManger: migrationManager)
-                            
-                        } else {
-                            try self.addPersistentStore(NSSQLiteStoreType, configuration: configuration, URL: storeURL, options: nil, migrationManger: nil)
-                        }
+                    } else {
+                        try self.addPersistentStore(NSSQLiteStoreType, configuration: configuration, URL: storeURL, options: nil, migrationManger: nil)
                     }
                 }
             }
-            NSNotificationCenter.defaultCenter().addObserver(self, selector: "handleContextDidSaveNotification:", name: NSManagedObjectContextDidSaveNotification, object: nil)
-            
-        } catch  let error as NSError {
-            
-            logError(tag) { "Failed to initialize: \(error.localizedDescription)" }
-            return nil
         }
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(GenericCoreDataStack.handleContextDidSaveNotification(_:)), name: NSManagedObjectContextDidSaveNotification, object: nil)
```

Comment:
Is listening for notifications still the recommended way to be informed about context changes?